lib/pbench/test/unit/server/test_user_management_cli.py

```diff
@@ -0,0 +1,251 @@
+import datetime
+
+import pytest
+import responses
+from click.testing import CliRunner
+
+from pbench.cli.server.user_management import (
+    USER_LIST_HEADER_ROW,
+    USER_LIST_ROW_FORMAT,
+    user_create,
+    user_delete,
+    user_list,
+    user_update,
+)
+
+
+class TestUserManagement:
+    USER_SWITCH = "--username"
+    PSWD_SWITCH = "--password"
+    PSWD_PROMPT = "Password: "
+    EMAIL_SWITCH = "--email"
+    FIRST_NAME_SWITCH = "--first-name"
+    LAST_NAME_SWITCH = "--last-name"
+    ROLE_SWITCH = "--role"
+    USER_TEXT = "test_user"
+    PSWD_TEXT = "password"
+    EMAIL_TEXT = "test@domain.com"
+    FIRST_NAME_TEXT = "First"
+    LAST_NAME_TEXT = "Last"
+    URL = "http://pbench.example.com/api/v1"
+    ENDPOINT = "/register"
```

Comment:
I believe that these are unused...you might as well remove them.  And, consider removing `responses`, as well.